integrations/transportation/uber.js

```diff
@@ -44,7 +44,10 @@ async function queryActivitiesFromOffset(offset, logger) {
     activityType: ACTIVITY_TYPE_TRANSPORTATION,
     carrier: 'Uber',
     datetime: new Date(d.start_time * 1000.0),
-    endDatetime: new Date(d.end_time * 1000.0),
+    endDatetime:
+      d.start_time === d.end_time
+        ? new Date((d.start_time + 3600) * 1000.0) // set a default ride time of an hour
```

Comment:
Isn't an hour quite a lot? The average Uber ride is around 6 miles and obviously depends on traffic, but in CPH that's closer to 20-30 mins :)

Reply:
haha yeah you're right :)